### PR TITLE
2/3: Update GObject Introspection to 1.60.0

### PR DESCRIPTION
**Pull request by Mno-hime**
Test suite requires updated Markdown module: https://github.com/OpenIndiana/oi-userland/pull/5019.

`libpeas` seems to need rebuild: https://github.com/OpenIndiana/oi-userland/pull/5018.

**Testing**

Rebuild:

- editor/pluma
- gnome/media/rhythmbox
- gnome/media/totem
- image/viewer/eom
- library/desktop/libpeas
- library/lua/lgi-52
- library/lua/lgi-53
- library/python/pygobject-3

Manually tested: totem, pluma, rhythmbox, eom.

ABI is stable: https://abi-laboratory.pro/index.php?view=timeline&l=gobject-introspection

Merge after Markdown PR is published: https://github.com/OpenIndiana/oi-userland/pull/5019.

### CODE REVIEW

**Review by alarcher — APPROVED** [review]

File: components/library/gobject-introspection/gobject-introspection.p5m [hunk 1]
```diff
@@ -73,6 +78,101 @@ file path=usr/lib/$(MACH64)/girepository-1.0/xfixes-4.0.typelib
 file path=usr/lib/$(MACH64)/girepository-1.0/xft-2.0.typelib
 file path=usr/lib/$(MACH64)/girepository-1.0/xlib-2.0.typelib
 file path=usr/lib/$(MACH64)/girepository-1.0/xrandr-1.3.typelib
+file path=usr/lib/$(MACH64)/gobject-introspection/giscanner/__init__.py
+file path=usr/lib/$(MACH64)/gobject-introspection/giscanner/_giscanner.a
```
Comment (pyhalov) [inline components/library/gobject-introspection/gobject-introspection.p5m]: _giscanner.a  is likely useless.

File: components/library/gobject-introspection/Makefile [hunk 2]
```diff
@@ -12,54 +12,63 @@
...
+	http://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/$(COMPONENT_VERSION_DL)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	http://live.gnome.org/GObjectIntrospection
+COMPONENT_LICENSE_FILE=	gobject-introspection.license
+COMPONENT_LICENSE=	GPLv2, LGPLv2
 
 include $(WS_TOP)/make-rules/prep.mk
 include $(WS_TOP)/make-rules/configure.mk
 include $(WS_TOP)/make-rules/ips.mk
 
+PYTHON_VERSION = 3.4
```
Comment (pyhalov) [inline components/library/gobject-introspection/Makefile:43]: Should we use 3.5 in new code where possible?

**Comment by pyhalov** [comment]
Please, rebase on oi/hipster.

File: components/library/gobject-introspection/Makefile (hunk 2 above)
Comment (pyhalov) [reply components/library/gobject-introspection/Makefile:43]: No, as it doesn't provide 32-bit libraries.

File: components/library/gobject-introspection/Makefile [hunk 3]
```diff
@@ -12,54 +12,63 @@
...
-  http://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/$(COMPONENT_VERSION_DL)/$(COMPONENT_ARCHIVE)
-COMPONENT_PROJECT_URL = http://live.gnome.org/GObjectIntrospection
+	http://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/$(COMPONENT_VERSION_DL)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	http://live.gnome.org/GObjectIntrospection
+COMPONENT_LICENSE_FILE=	gobject-introspection.license
+COMPONENT_LICENSE=	GPLv2, LGPLv2
 
 include $(WS_TOP)/make-rules/prep.mk
 include $(WS_TOP)/make-rules/configure.mk
 include $(WS_TOP)/make-rules/ips.mk
```
Comment (AndWac) [inline components/library/gobject-introspection/Makefile]: Please change the includes such that they make use of $(WS_MAKE_RULES).

**Comment by pyhalov** [comment]
Couldn't fix it in this PR, created and merged https://github.com/OpenIndiana/oi-userland/pull/5138

**Pull request closed without merging**